src/main/webapp/client.js

```diff
@@ -29,23 +25,23 @@ function listAllCharities() {
     dataElement.innerHTML = "";
     charities.forEach((charity) => {
       const div = document.createElement("div");
+      div.className = "card";
 
       const header = document.createElement("h3");
       header.textContent = charity.name;
 
       const contents = document.createElement("p");
       contents.textContent = charity.description;
 
-      const address = document.createElement("p");
-      address.textContent = charity.address;
+      const location = document.createElement("p");
+      location.textContent = charity.location;
 
       div.appendChild(header);
       div.appendChild(contents);
```

Comment:
Maybe we can rename this variable from `contents` to `description` for consistency and more clarity?

Reply:
agree with the renaming